Sources/MongoSwift/ConnectionString.swift

```diff
@@ -40,6 +40,24 @@ internal class ConnectionString {
             }
         }
 
+        let invalidLocalThresholdError = MongoError.InvalidArgumentError(
+            message: "localThresholdMS must be between 0 and \(Int32.max)"
```

Comment:
can this include the value the user set?

Reply:
done